test/clients/messaging-api/Api/MessagingApiApiTest.php

```diff
@@ -50,8 +51,38 @@ public function testGetFollowers(): void
                 body: json_encode(['userIds' => ["Uaaaaaaaa...", "Ubbbbbbbb...", "Ucccccccc..."], 'next' => "yANU9IA.."]),
             ));
         $api = new MessagingApiApi($client);
-        $followers = $api->getFollowers(limit:  99);
+        $followers = $api->getFollowers(limit: 99);
         $this->assertEquals(["Uaaaaaaaa...", "Ubbbbbbbb...", "Ucccccccc..."], $followers->getUserIds());
         $this->assertEquals("yANU9IA..", $followers->getNext());
     }
-}
+
+    public function testIssueStatelessChannelToken(): void
+    {
+        $client = Mockery::mock(ClientInterface::class);
+        $client->shouldReceive('send')
+            ->with(
+                Mockery::on(function (Request $request) {
+                    $this->assertEquals('POST', $request->getMethod());
+                    $this->assertEquals('https://api.line.me/oauth2/v3/token', (string)$request->getUri());
+                    $this->assertEquals('grant_type=client_credentials&client_id=1234&client_secret=clientSecret', (string)$request->getBody());
+                    return true;
+                }),
+                []
+            )
+            ->once()
+            ->andReturn(new Response(
+                status: 200,
+                headers: [],
+                body: json_encode(['access_token' => 'accessToken', 'expires_in' => 30, 'token_type' => 'Bearer',]),
+            ));
+        $api = new ChannelAccessTokenApi($client);
+        $response = $api->issueStatelessChannelToken("client_credentials",
```

Comment:
How about explicitly using required arguments to work tests even if new optional argument is required?
`issueStatelessChannelToken(grantType: "client_credentials", clientId: "1234", clientSecret: "channelSecret")`

Reply:
Yes, I'll do that. In fact, I think that this is also the more common use case for users.